OrdinalDate.rb

```diff
@@ -0,0 +1,56 @@
+class Ordinalize
```

Comment:
<img width="824" alt="Screenshot 2022-04-01 at 09 54 33" src="https://user-images.githubusercontent.com/6873183/161220195-2ea4175b-6c84-4f26-9b75-7f2a884227c2.png">

Reply:
it does not work

Reply:
<img width="793" alt="Screenshot 2022-04-01 at 09 55 58" src="https://user-images.githubusercontent.com/6873183/161220445-efca92b0-c4ab-4bb0-868a-1df4ae373aac.png">